controllers/user_controller_test.go

```diff
@@ -73,6 +74,7 @@ func prepareTest(t *testing.T, initObjs ...client.Object) client.WithWatch {
 	utilruntime.Must(orgv1.AddToScheme(scheme))
 	utilruntime.Must(controlv1.AddToScheme(scheme))
 	utilruntime.Must(billingv1.AddToScheme(scheme))
+	utilruntime.Must(userv1.AddToScheme(scheme))
 
 	return fake.NewClientBuilder().
 		WithScheme(scheme).
```

Comment:
Nitpick: This probably shouldn't be in `user_controller_test.go` anymore as it's used by a lot of different controller tests now :sweat_smile:

Reply:
fair enough 😅 moved to its own file